akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -183,7 +186,7 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
   /** INTERNAL API. */
   override protected[akka] def aroundPreRestart(reason: Throwable, message: Option[Any]): Unit = {
     try {
-      internalStash.unstashAll()
+      unstashInternally(all = true)
```

Comment:
I wouldn't think of it as `unstashAll` being encapsulated but the case of conditional `unstashAll` which this isn't. Not important though, if you feel strongly about this, let it be as is.